### ISSUE TITLE
Caryssia Moonhunter not working

### ISSUE THREAD

**Issue by kevinwiegman** [elysium-project/bug-tracker#229]
The Tribal Leatherworking trainer "Caryssia Moonhunter" in Ferales is not working (for me). After clicking "Train me", nothing happends.

I tried reloading the interface, restarting wow and disabling addons. Nothing worked.

**Comment by Navashed** [comment]
Confirmed.

Caryssia Moonhunter (NPC 7870) has the "Train me!" dialogue option for anyone with Leatherworking, but unless the player knows Tribal Leatherworking (Spell 10660) she has nothing to train and the dialogue immediately disappears.

She should not offer to train anyone who does not have Tribal Leatherworking.

Also, while investigating this issue I discovered that she offers her quest, Tribal Leatherworking (Quest 5143), without requiring any pre-reqs. It is supposed to require Master of the Wild Leather (Quest 2853).
